### ISSUE TITLE
The widget is not disposed probably 

### ISSUE THREAD

**Issue by a-eniou-pvotal** [Mindinventory/clock_loader#1]
I tried your library and after it got disappeared I got the following warning

```
════════ Exception caught by widgets library ═══════════════════════════════════
The following assertion was thrown while finalizing the widget tree:
_ClockSimplePathState#3d015(tickers: tracking 12 tickers) was disposed with an active Ticker.

_ClockSimplePathState created a Ticker via its TickerProviderStateMixin, but at the time dispose() was called on the mixin, that Ticker was still active. All Tickers must be disposed before calling super.dispose().

Tickers used by AnimationControllers should be disposed by calling dispose() on the AnimationController itself. Otherwise, the ticker will leak.

The offending ticker was: _WidgetTicker(created by _ClockSimplePathState#3d015)
The stack trace when the _WidgetTicker was actually created was:
package:flutter/…/widgets/ticker_provider.dart:202
#4      new AnimationController
package:flutter/…/animation/animation_controller.dart:246
#5      _ClockSimplePathState.fillAnimationController
package:clock_loader/src/**clock_loader_particles.dart:73**
#6      _ClockSimplePathState.initState.<anonymous closure>

```

**Comment by a-eniou-pvotal** [comment]
**fillAnimationController** is called again after disposed

**Comment by sagarfaldu17** [comment]
closes #2, i have done with the disposing as well as reusing issue part, also tested the scenario by disposing it several time to check the behaviour is fine, and the latest changes is merged in master.

**Comment by sagarfaldu17** [comment]
I have done with the disposing as well as reusing issue part, also tested the scenario by disposing it several time to check the behaviour is fine, and the latest changes is merged in master.
